src/main/java/io/wisetime/idea/branch/settings/SettingsState.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021 Practice Insight Pty Ltd. All Rights Reserved.
+ */
+
+package io.wisetime.idea.branch.settings;
+
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.components.PersistentStateComponent;
+import com.intellij.openapi.components.State;
+import com.intellij.openapi.components.Storage;
+import com.intellij.openapi.wm.impl.IdeFrameDecorator;
+import com.intellij.util.xmlb.XmlSerializerUtil;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * @author vadym
+ */
+@State(
+    name = "io.wisetime.idea.branch.settings.SettingsState",
+    storages = @Storage("WTBranchWindowTitleSettings.xml")
+)
+public class SettingsState implements PersistentStateComponent<SettingsState> {
+
+  public boolean appendToWindowTitle = true;
+  public boolean appendToProjectName = IdeFrameDecorator.isCustomDecorationAvailable();
```

Comment:
By default project name is changed only on Windows, other OS will have previous behaviour - only win title changes

Reply:
Or to play it safe we could disable it by default for all OS and maybe update screenshots?

Reply:
Is it a valid state that both appendToWindowTitle and appendToProjectName are true?

Reply:
Yes, we change both. On windows user don't see window title. On mac/linux/apps that read window title, can see custom frame decoration view.